site/quickstart.md

```diff
@@ -37,6 +37,25 @@ $ docker pull store/oracle/weblogic:12.2.1.3
 f.	Then patch the WebLogic image according to these [instructions](https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-patch-wls-for-k8s),
     and copy the image to all nodes in your cluster, or put it in a Docker registry that your cluster can access.
 
+g.  Grant the Helm service account the `cluster-admin` role:
```

Comment:
Should this be a step by itself? Asking only because the step title is about getting images. I am ok either way.

Reply:
Technically, yes. You are right about that. It would involve me manually re-numbering all the steps to accommodate the change and it adds to the number of steps in the "quick" start. I might do it at a later date.